docker/dev.dockerfile

```diff
@@ -75,6 +75,7 @@ ARG pip_dependencies=' \
       attrs \
       contextlib2 \
       dataclasses \
+      flax \
```

Comment:
For historical reason, there is a separate docker file for the lingvo+JAX here.
https://github.com/tensorflow/lingvo/blob/master/lingvo/jax/pip_package/build.Dockerfile

Could you revert this change and try to use this docker file instead?

Reply:
Done. 

I also ran into a problem with "//third_party/py/einshape" in jax/layers/BUILD.

Separately, I tried to build docker and ran into a different error:
```
% docker build --tag tensorflow:lingvo $(test "$LINGVO_DEVICE" = "gpu" && echo "--build-arg base_image=nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04") - < "$LINGVO_DIR/lingvo/jax/pip_package/build.Dockerfile"
...
#11 7.361 E: Unable to locate package python3.7
#11 7.361 E: Couldn't find any package by glob 'python3.7'
#11 7.361 E: Couldn't find any package by regex 'python3.7'
#11 7.361 E: Unable to locate package python3.7-distutils
#11 7.361 E: Couldn't find any package by glob 'python3.7-distutils'
#11 7.361 E: Couldn't find any package by regex 'python3.7-distutils'
#11 7.361 E: Unable to locate package python3.8
#11 7.361 E: Couldn't find any package by glob 'python3.8'
#11 7.361 E: Couldn't find any package by regex 'python3.8'
#11 7.361 E: Unable to locate package python3.8-distutils
#11 7.361 E: Couldn't find any package by glob 'python3.8-distutils'
#11 7.361 E: Couldn't find any package by regex 'python3.8-distutils'
#11 7.361 E: Unable to locate package python3.9-distutils
#11 7.361 E: Couldn't find any package by glob 'python3.9-distutils'
#11 7.361 E: Couldn't find any package by regex 'python3.9-distutils'
------
executor failed running [/bin/sh -c apt-get update && apt-get install -y python3.7 python3.7-distutils python3.8 python3.8-distutils python3.9 python3.9-distutils]: exit code: 100
```

Reply:
Sorry about this :(

Let's merge your request first, since this fixes real issues.
I will try to investigate your other issues.

Thanks for the changes!